src/Markdown/Editor/Impl/ContentType/MdContentTypeDefinition.cs

```diff
@@ -6,11 +6,9 @@
 
 namespace Microsoft.Markdown.Editor.ContentTypes {
     public sealed class MdContentTypeDefinition {
-        public const string LanguageName = "Markdown";
-        public const string ContentType = "Markdown";
-        public const string FileExtension1 = ".md";
-        public const string FileExtension2 = ".markdown";
-        public const string FileExtension3 = ".rmd";
```

Comment:
Not really. 99% people use RMD. We never fully supported plain markdown and generally people install Web Extensions to edit it. Dev14 U2+ also provides textmate grammar so MD is colorized.